src/content/reference/cli.mdx

```diff
@@ -196,6 +196,7 @@ If the images in the `build` section don't exist, `okteto deploy` automatically
 If the repositories in the `dependencies` section haven't been deployed yet, `okteto deploy` deploys these dependencies automatically.
 
 The command is executed relative to the path where the manifest is located. This means, that if the manifest is at any subfolder, the pipeline will use the context of the subfolder where the manifest is defined.
+Executed commands will be executed using bash unless the `--no-bash` flag has been specified.
```

Comment:
I think this might look better as a quote/note like we do for other comment and suggestions